frontend/firestore.rules

```diff
@@ -3,7 +3,7 @@ service cloud.firestore {
   match /databases/{database}/documents {
     match /{document=**} {
       allow read, write: if
-          request.time < timestamp.date(2023, 8, 31);
+          request.time < timestamp.date(2023, 09, 22);
```

Comment:
It should be noted that we don't suffer from [zero prefixed octal numbers](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Numbers_and_dates) in JavaScript here as `9` is not an octal digit.